src/components/utils/test/conditional_variable_test.cc

```diff
@@ -38,32 +38,45 @@
 #include "utils/lock.h"
 #include "utils/macro.h"
 #include "utils/conditional_variable.h"
+#include "utils/threads/thread.h"
+#include "utils/threads/thread_delegate.h"
 
 namespace test {
 namespace components {
 namespace utils_test {
 
 class ConditionalVariableTest : public ::testing::Test {
  public:
-  ConditionalVariableTest() : test_value_("initialized"), counter_(0) {}
-  void check_counter();
-  void task_one();
+  ConditionalVariableTest() {}
+  static void check_counter();
```

Comment:
@OHerasym  unclear method purpose. Please Add doxygen comment and change to CamelCase